scripts/lc-builds/blueos-EA_nvcc9_clang6.0.0.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+i#!/usr/bin/env bash
```

Comment:
Typo

Reply:
Fixed. God catch! We really need to get our CI set up on the LC....

Reply:
We are making some progress... 😄